src/agent/onefuzz-task/src/tasks/fuzz/libfuzzer/dotnet.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use anyhow::Result;
+use async_trait::async_trait;
+use onefuzz::libfuzzer::LibFuzzer;
+use tokio::process::Command;
+
+use crate::tasks::fuzz::libfuzzer::common;
+
+#[cfg(target_os = "linux")]
+const LIBFUZZER_DOTNET_PATH: &str =
+    "/onefuzz/third-party/dotnet-fuzzing-linux/libfuzzer-dotnet/libfuzzer-dotnet";
+
+#[cfg(target_os = "windows")]
+const LIBFUZZER_DOTNET_PATH: &str =
+    "/onefuzz/third-party/dotnet-fuzzing-windows/libfuzzer-dotnet/libfuzzer-dotnet.exe";
```

Comment:
Shouldn't this be `C:/onefuzz/...`?

Reply:
As-is, the path is relative to the current drive (the drive of the current working directory). In this context, that means the drive of the current working dir of the `onefuzz-task` process. We can either assume that these drives are the same, or that it should be `C:`. IMO the former is a safer assumption. In reality, it doesn't matter, and if we ever needed to stop hardcoding this value, we'd have to make it so implementers of `LibFuzzerTaskType` can have their own data fields. WDYT is clearest?

Reply:
Oh I didn't know about that behavior, I thought it would've been an invalid path. I think leaving it as it is is good then!